pkgs/tools/misc/nyancat/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  name = "nyancat-${version}";
```

Comment:
Since recent times, using `pname = "nyancat"` works, which will automatically set `name` too

Reply:
Ah, that's nice.